Exiled.Events/Exiled.Events.csproj

```diff
@@ -35,4 +35,8 @@
     <Reference Include="UnityEngine.PhysicsModule" HintPath="$(EXILED_REFERENCES)\UnityEngine.PhysicsModule.dll" Private="false" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+
```

Comment:
Remove these useless changes. Can't figure out why Visual Studio did this, because it's a new project style which doesn't include a Properties folder with a AssemblyInfo file, maybe you didn't update .csproj files?